integration-tests/load/vrfv2/vrfv2_test.go

```diff
@@ -254,65 +165,158 @@ func TestVRFV2Performance(t *testing.T) {
 			Interface("Fulfilment Count", fulfilmentCount).
 			Msg("Final Request/Fulfilment Stats")
 	})
-
 }
 
-func cancelSubsAndReturnFunds(ctx context.Context, subIDs []uint64, l zerolog.Logger) {
-	for _, subID := range subIDs {
-		l.Info().
-			Uint64("Returning funds from SubID", subID).
-			Str("Returning funds to", eoaWalletAddress).
-			Msg("Canceling subscription and returning funds to subscription owner")
-		pendingRequestsExist, err := vrfContracts.CoordinatorV2.PendingRequestsExist(ctx, subID)
-		if err != nil {
-			l.Error().Err(err).Msg("Error checking if pending requests exist")
-		}
-		if !pendingRequestsExist {
-			_, err := vrfContracts.CoordinatorV2.CancelSubscription(subID, common.HexToAddress(eoaWalletAddress))
-			if err != nil {
-				l.Error().Err(err).Msg("Error canceling subscription")
-			}
-		} else {
-			l.Error().Uint64("Sub ID", subID).Msg("Pending requests exist for subscription, cannot cancel subscription and return funds")
-		}
+func TestVRFV2BHSPerformance(t *testing.T) {
+	l := logging.GetTestLogger(t)
+
+	testType, err := tc.GetConfigurationNameFromEnv()
+	require.NoError(t, err)
+	testConfig, err := tc.GetConfig(testType, tc.VRFv2)
+	require.NoError(t, err)
+
+	testReporter := &testreporters.VRFV2TestReporter{}
+	vrfv2Config := testConfig.VRFv2
+
+	cfgl := testConfig.Logging.Loki
+	lokiConfig := wasp.NewLokiConfig(cfgl.Endpoint, cfgl.TenantId, cfgl.BasicAuth, cfgl.BearerToken)
```

Comment:
What is this used for? Are you interacting with Loki somehow?

Reply:
yes, we send consumer stats to [Grafana](https://grafana.ops.prod.cldev.sh/d/WaspVRFv2Plus/waspvrfv2plus?orgId=1&refresh=5s&var-go_test_name=TestVRFV2PlusPerformance&var-gen_name=performance&var-branch=vrfv2Plus_healthcheck&var-commit=vrfv2Plus_healthcheck&var-call_group=All).